### PR TITLE
Add test to create multiple namespaces with multiple regions

### PR DESCRIPTION
**Pull request by harish-24**
Add test to create multiple namespaces with multiple regions

Signed-off-by: Harish <harish@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by harish-24** [comment]
```
 (7/9) memory/ndctl.py:NdctlTest.test_multiple_namespaces_multiple_region: CANCEL: Test not applicable without multiple regions (0.08 s)
```
```
 (7/9) memory/ndctl.py:NdctlTest.test_multiple_namespaces_multiple_region: PASS (2.99 s)
```

File: memory/ndctl.py [hunk 1]
```diff
@@ -201,6 +202,11 @@ def setUp(self):
                 self.cancel('%s is needed for the test to be run' % pkg)
         self.opt_dict = {'-B': 'provider',
                          '-D': 'dev', '-R': 'dev', '-N': 'dev'}
+        if "multiple_ns_multiple_region" in self.name.name:
```
Comment (narasimhan-v) [inline memory/ndctl.py]: Can't this be moved into `test_multiple_ns_multiple_region()` ?

File: memory/ndctl.py [hunk 2]
```diff
@@ -279,33 +285,67 @@ def test_multiple_namespaces_region(self):
...
+        self.enable_region()
+        regions = self.get_json(short_opt='-R')
+        self.disable_namespace()
+        self.destroy_namespace()
+        for region in regions:
+            self.log.info("Using %s for muliple namespaces", region)
+            size = self.get_json_val(self.get_json(
+                long_opt='-r %s' % region)[0], 'size')
+            self.log.info("Creating %s namespaces", self.cnt)
+            if 'Hash' in genio.read_file('/proc/cpuinfo').rstrip('\t\r\n\0'):
```
Comment (narasimhan-v) [inline memory/ndctl.py]: This check is being done twice in this script. Better to move it into a helper function.

**Comment by harish-24** [comment]
@narasimhan-v Pushed the changes. Thanks.

**Pull request merged**